catalog/opensearch/catalog-opensearch-endpoint/src/test/java/org/codice/ddf/opensearch/endpoint/query/OpenSearchQueryTest.java

```diff
@@ -668,6 +718,74 @@ public void testPolygonSpatialFilter() {
     }
   }
 
+  @Test
+  public void testGeometrySpatialFilter() {
+    OpenSearchQuery query = new OpenSearchQuery(0, 10, "relevance", "desc", 30000, FILTER_BUILDER);
+    query.addGeometrySpatialFilter(GEOMETRY_WKT);
+    Filter filter = query.getFilter();
+    assertThat(filter, notNullValue());
+    Intersects intersects = (Intersects) filter;
+    Literal literalWrapper = (Literal) intersects.getExpression2();
+    Object geometryExpression = literalWrapper.getValue();
+    assertThat(geometryExpression, instanceOf(GeometryImpl.class));
+    com.vividsolutions.jts.geom.Geometry polygon =
```

Comment:
do we need the full package name?

Reply:
yes, the default import is org.opengis.geometry.Geometry